src/central_limit_theorem/CentralLimitGraph.jsx

```diff
@@ -38,23 +38,22 @@ export class CentralLimitGraph extends Component {
         this.generatePopulation = this.generatePopulation.bind(this);
         this.handleGeneratePopulation = this.handleGeneratePopulation
             .bind(this);
+        this.handleSampleForm = this.handleSampleForm.bind(this);
         this.handleResetPopulation = this.handleResetPopulation.bind(this);
```

Comment:
This function can be removed since it's unused now.

Reply:
Got it. I pulled it.